packages/node_modules/@webex/plugin-meetings/src/meetings/request.js

```diff
@@ -48,6 +48,8 @@ export default class MeetingRequest extends StatelessWebexPlugin {
         })
         .catch((error) => {
           LoggerProxy.logger.error(`MeetingRequest->determineRedirections# failed to get locus details from url: ${url}, reason: ${error}`);
+
```

Comment:
I take it this wasn't suppose to be an acceptable error. Was there a reason it was ignored before?